src/oss/confluent.sh

```diff
@@ -1697,21 +1716,22 @@ consume_usage() {
     fi
 
     cat <<EOF
-Usage: ${command_name} consume <topicname> ] [--value-format avro] [other optional args]
+Usage: ${command_name} consume <topicname> ] [--value-format avro] [--cloud] [other optional args]
 
 Description:
     Consume from Kafka topic specified by <topicname>.
 
     By default, this command consumes from the Kafka cluster on the localhost.
     To consume from another Kafka cluster, set the '--bootstrap-server' argument.
+    To consume Confluent Cloud, set the '--cloud' argument (assumes Confluent Cloud configuration file at $HOME/.ccloud/config and topic already created in Confluent Cloud).
 
     By default, this command reads non-Avro data.
     To read Avro data, specify '--value-format avro'
 
 Examples:
     confluent consume mytopic1 --value-format avro --from-beginning
 
-    confluent consume mytopic1 --from-beginning
```

Comment:
same as with produce above, we probably want to retain the original example with minimal parameters that refers to a local cluster.

Reply:
In both cases I'd have this example first too, to align with the order mentioned in the description.

Reply:
Agree